sql/core/src/test/resources/sql-tests/results/describe.sql.out

```diff
@@ -561,7 +561,7 @@ struct<plan:string>
 -- !query output
 == Physical Plan ==
 Execute DescribeColumnCommand
-   +- DescribeColumnCommand `default`.`t`, [b], false
+   +- DescribeColumnCommand `default`.`t`, [spark_catalog, default, t, b], false
```

Comment:
Now that attribute is resolved, a fully qualified name is passed to `DescribeColumnCommand`